### DISCUSSION TITLE
Testing and looking for bugs on sublayer.art page

### DISCUSSION THREAD

**Discussion by SasoLithops** [ringdao-community-guild/discussion#190] [General]
Hi!

I tested with Metamask on Chrome. Please let me know if any other needed.

Here are my findings:

- Under TRADE tab TO LAUNCHPAD doesn't take you to launchpad page. Probably planned for now.
- When listing I can not use decimal number. I tried listing with 2.5 RING but get Can not convert 2.5 to BigInt. 2,5 doesn't work.  
- When I put my NFTs on sell, UNLIST doesn't work for me. I just click UNLIST but nothing happens uder both LISTED and COLLECTIONS tabs.
- I also recommend when listing NFTs they should be only under LISTED not also under COLLECTIONS.
- Instead of COLLECTIONS I recommend TOKENS or MY NFTs
- What does star and fire logo mean in TRADE page? You can not click them alone. Are they for favorite and hot collections?
- Clicking on SUBLAYER in top left corner should take you to main page where you guys and some hot collections are introduced imho.
- Where can artists launch its collection?

**Comment by hackfisher** [comment]
@SasoLithops Thanks for the testing, invited you to the communication group with sublayer.art

**Reply by SasoLithops** [reply]
It is my honor to help testing for anyone from RING DAO family.
Is this group a TG group? I am already a member there😊

**Comment by leo-xinwang** [comment]
@SasoLithops  Thanks a lot for your testing. We have fixed major issues and recorded as below.

- Under TRADE tab TO LAUNCHPAD doesn't take you to launchpad page. Probably planned for now.
Ans: this will be implemented as an update after 1.0
- When listing I can not use decimal number. I tried listing with 2.5 RING but get Can not convert 2.5 to BigInt. 2,5 doesn't work.
Ans: test it on a mouse NFT on trading page, 2.5 RING is working
- When I put my NFTs on sell, UNLIST doesn't work for me. I just click UNLIST but nothing happens uder both LISTED and COLLECTIONS tabs.
Ans: fixed
- I also recommend when listing NFTs they should be only under LISTED not also under COLLECTIONS.
Ans: current design as collections page is showing all your NFTs
- Instead of COLLECTIONS I recommend TOKENS or MY NFTs
Ans: will improve the wording
- What does star and fire logo mean in TRADE page? You can not click them alone. Are they for favorite and hot collections?
Ans: this will be implemented as an update after 1.0
- Clicking on SUBLAYER in top left corner should take you to main page where you guys and some hot collections are introduced imho.
Ans: fixed
- Where can artists launch its collection?
Ans: same as the 1st issue, this will be implemented as an update after 1.0

**Reply by SasoLithops** [reply]
I was glad to help. Looking forward to updates🏆 Feel free to tag me if more testing needed.
